addOns/help/src/main/javahelp/contents/releases/2_8_0.html

```diff
@@ -89,6 +89,9 @@ <h3>Client Certificate Handling</h3>
 configuration file (so that they apply for subsequent ZAP use).
 </p>
 
+<H2>Source Code Restructuring</H2>
+The ZAP repositories have all been migrated to use Gradle from Ant. The source code is also now auto formatted for consistency.
```

Comment:
Should we be more specific about the formatting? This almost makes it sound like contributors don’t need to worry about it at all.

Reply:
Updated